jdaviz/configs/default/plugins/gaussian_smooth/gaussian_smooth.py

```diff
@@ -88,15 +88,19 @@ def user_api(self):
 
     @observe("dataset_selected", "dataset_items", "stddev", "mode_selected")
     def _set_default_results_label(self, event={}):
-        label_comps = []
-        if hasattr(self, 'dataset') and (len(self.dataset.labels) > 1 or self.app.config == 'mosviz'):  # noqa
-            label_comps += [self.dataset_selected]
-        if self.config == "cubeviz":
-            label_comps += [f"{self.mode_selected.lower()}-smooth"]
+        '''Generate a label and set the results field to that value'''
+        if (hasattr(self, 'dataset') and (len(self.dataset.labels) >= 1) or self.app.config == 'mosviz'):  # noqa
+            dataset = self.dataset_selected
         else:
-            label_comps += ["smooth"]
-        label_comps += [f"stddev-{self.stddev}"]
-        self.results_label_default = " ".join(label_comps)
+            dataset = ''
```

Comment:
won't this result in an empty space at the beginning of the formatted label?

Reply:
I traced the behavior, and the path is strange... let me try to explain (in Cubeviz):

When a user loads their data (which happens usually at the start of their environment), this code is triggered 3 times: (1) When the config is initialized, (2) when data is added to the flux viewer, and (3) adding the autocollapsed spectrum to the spectrum viewer. The first two times, `self.dataset` is empty, which, yes, sets the empty string and results in an empty space at the start. But the third time, the dataset is registered with the plugin and it properly populates `self.dataset` which then fills the text field automatically. 

All of this is transparent to the user, since it's all happening at initialization time. The only time the user might find this scenario is if they start the app and open the plugin, but haven't loaded any data into the tool, in which case they can't even smooth to begin with.

To put it another way, testing against both Cubeviz and Specviz, the only time I could see the empty space is when there is no data in the tool at all. In which case trying to smooth results in a traceback (unsurprisingly). Once I loaded data, the string formatted properly.

Reply:
ok, that makes sense.  I realize it's out of scope of what was asked here, but how do you feel about just returning an empty string in that case (for the whole label)?  That might be easier to follow in the future.  Or alternatively an inline comment in the else explaining that it will be immediately rewritten once data is loaded?

Reply:
All the above being said, I think having _something_ in the field is also a good "but what if" backup scenario. In the event something goes off the rails and a dataset is _somehow_ created with these strings, at least it's descriptive enough to know _kind of_ what it is. I'm nervous with it being entirely blank. Thoughts?

Reply:
ok, that's a good point, I think this is probably fine for now and if we ever run into a case where its a problem we can update it then.  Thanks!

Reply:
Randomly came up with a potential solution to your point here in https://github.com/spacetelescope/jdaviz/pull/1973/commits/33f577a5ab85d1244d5eccaf13f350b758273bf8. This should avoid the leading whitespace